### PR TITLE
intent updated

### PR DESCRIPTION
**Pull request by SabalM**

### CODE REVIEW

File: intents.json [hunk 1]
```diff
@@ -211,9 +214,9 @@
                 "What's the forecast for the week"
             ],
             "responses": [
-                "Sure! What city would you like the weather forecast for?",
-                "I'd be happy to check the weather. Could you please tell me the city?",
-                "Great! To provide the weather forecast, I just need to know the city name. What city are you interested in?"
+                "The current temperature in {city} is {temperature} degrees Celsius. And, currently the weather is {description}.",
```
Comment (LuluW8071) [inline intents.json]: Change the temperature responses to previous

```json
"responses": [
                "Sure! What city would you like the weather forecast for?",
                "I'd be happy to check the weather. Could you please tell me the city?",
                "Great! To provide the weather forecast, I just need to know the city name. What city are you interested in?"
]
```

File: intents.json [hunk 2]
```diff
@@ -259,13 +269,26 @@
                 "Personally, I find my creator 'Shashin' very inspiring. I might not be very fair though... since half of me was also built by 'Saugat', 'Sabal' and 'Neema'."
             ]
         },
+        {
```
Comment (LuluW8071) [inline intents.json]: Remove the greetreply tag as well 
Since `greeting` tag already exists

**Review by LuluW8071 — CHANGES_REQUESTED** [review]
Need some changes in `weather` and `greetreply` tag  @SabalM

File: intents.json (hunk 2 above)
Comment (SabalM) [reply intents.json]: greetreply removed

File: intents.json (hunk 1 above)
Comment (SabalM) [reply intents.json]: changed temperature response to previous

**Review by LuluW8071 — APPROVED** [review]
👍🏻

**Pull request merged**